bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -1411,6 +1411,6 @@ public void restoreScienceDetails(final SPInstObsComp oldData) {
 
     @Override
     public ScienceAreaGeometry getVignettableScienceArea() {
-        return new F2ScienceAreaGeometry(this);
+        return F2ScienceAreaGeometry$.MODULE$;
```

Comment:
We have added in a few places an `instance` val to the object to avoid the `$MODULE$` call in java